plugins/scaffolder-react/src/api/types.ts

```diff
@@ -45,15 +45,23 @@ export type ScaffolderTask = {
 };
 
 /**
- * A single action example
+ * A single example
  *
  * @public
  */
-export type ActionExample = {
-  description: string;
+export type Example = {
+  description?: string;
   example: string;
+  notes?: string;
 };
 
+/**
+ * A single action example
+ *
+ * @public
+ */
+export type ActionExample = Example;
```

Comment:
Deprecate this type? Maybe `Example` should also be `ExtensionExample` or something?

Reply:
I am not opposed to enhancing the name of `Extension` in some way, but there seem to some issues:

 * I extracted the example rendering to a reusable component which operates on this basic structure
 * I of course retrofitted `ActionExample` for purposes of backward compatibility, but if `ActionExample` equals/extends `ExtensionExample` that seems to imply that actions are extensions, which I don't personally feel is accurate